R/mboostLSS.R

```diff
@@ -159,16 +198,27 @@ mboostLSS_fit <- function(formula, data = list(), families = GaussianLSS(),
         ## update value of nuisance parameters in families
         for (k in mods[-j]){
             if (!is.null(fit[[k]]))
-                assign(names(fit)[k], fitted(fit[[k]], type = "response"),
+                assign(names(fit)[k], families[[k]]@response(mboost:::fitted.mboost(fit[[k]])),
                        environment(families[[j]]@ngradient))
         }
         ## use appropriate nu for the model
         control$nu <- nu[[j]]
         ## <FIXME> Do we need to recompute ngradient?
-        fit[[j]] <- do.call(fun, list(formula[[names(families)[[j]]]],
-                                      data = data, family = families[[j]],
-                                      control=control, weights = w,
-                                      ...))
+        if(funchar != "FDboost"){
+          fit[[j]] <- do.call(fun, list(formula[[names(families)[[j]]]],
+                                        data = data, family = families[[j]],
+                                        control = control, weights = w,
+                                        ...))
+        }else{
+          fit[[j]] <- do.call(fun, list(formula[[names(families)[[j]]]],
+                                        timeformula = timeformula[[names(families)[[j]]]],
+                                        data = data, family = families[[j]],
+                                        control = control, weights = w, 
+                                        # always use scalar offset, as offsets are treated within the Family
+                                        offset = "scalar",
```

Comment:
Yes, but only to this special one. In `FDboost()`, the default `offset = NULL` yileds an offset that is a smooth function over time, the call `offset = "scalar"` gives the offset as in mboost, i.e., one global constant. 
In `FDboostLSS()`, the offset is supposed to be a constant value for each distribution parameter, which we get by this call.